### DISCUSSION TITLE
接入子应用时一直报SyntaxError: Unexpected token 'export'

### DISCUSSION THREAD

**Discussion by lower-greenHand** [jd-opensource/micro-app#1344] [General]
接入子应用时控制台一直报SyntaxError: Unexpected token 'export'，子应用打好包也是es6方式，但是主应用就是访问不了

**Comment by bailicangdu** [comment]
切换到iframe沙箱看看，https://micro-zoe.github.io/micro-app/docs.html#/zh-cn/configure?id=iframe
